recipes/tvb-data/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: tvb-data
+  version: "1.5.4"
+
+source:
+  md5: 2b0518fc64a630146f93a8819be30b9a
+  fn: 'tvb-data-1.5.4.tar.gz'
```

Comment:
Like `gdist`, this should be `tvdb-data-{{ version }}.tar.gz` (or `{{ name }}-{{ version }}.tar.gz`).

Reply:
Not adding comments for all of them, but all of the `fn` and `url` properties should use templates like this. Makes it much easier to update in the future.